src/devices/systemReady/SystemReady_nws_yarp.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2006-2021 Istituto Italiano di Tecnologia (IIT)
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "SystemReady_nws_yarp.h"
+#include <yarp/os/LogComponent.h>
+#include <yarp/os/LogStream.h>
+
+YARP_LOG_COMPONENT(SYSTEMREADY_NWS_YARP, "yarp.devices.SystemReady_nws_yarp")
+
+SystemReady_nws_yarp::SystemReady_nws_yarp()
+{
+}
+
+bool SystemReady_nws_yarp::open(yarp::os::Searchable &config)
+{
+    if (config.check("PORT_LIST")) {
+        yarp::os::Bottle port_list = config.findGroup("PORT_LIST");
+        // skips the first one since it is PORT_LIST
```

Comment:
It is also possible to use [`yarp::os::Bottle::tail`](http://www.yarp.it/latest/classyarp_1_1os_1_1Bottle.html#a9bac31d80e659b650cb643449b93d171) as a shortcut.